specs/transformations/BCELoss.transformation.yaml

```diff
@@ -32,5 +33,5 @@ outputs:
     - name: loss
       data_type: float32
       data_range: [0, 1]
-      # axes: null   we don't care about the axes here, so this should be optional
+      axes: "c"
```

Comment:
I think that the BCELoss will just return a scalar, so having the `axes: "c"` is not really correct.

Reply:
true, removed axes and changed shape to
```
shape: []
```